react-ui/src/pages/submission/SubmissionForm.jsx

```diff
@@ -498,23 +498,33 @@ function SubmissionForm({
                 [classes.previewDescription]: previewDescription,
               })}
               label={`Description${previewDescription ? ' Preview' : ''}`}
-              helperText={
-                formState.description.error || (
-                  <div>
-                    This should be a 1-4 sentence description of your flag that
-                    explains any design choices you made. You can use
+              helperText={(
+                <div>
+                  {formState.description.error ? (
+                    <Typography display="inline">{formState.description.error}</Typography>
+                  ) : (
+                    <Typography display="inline">
+                      Explains any design choices you made. You can use
+                      {' '}
+                      <ExternalLink color="secondary" href="https://www.reddit.com/wiki/markdown">
+                        Reddit Markdown
+                      </ExternalLink>
+                      {' '}
+                      in this field.
+                    </Typography>
+                  )}
+                  <Typography display="inline">
```

Comment:
There's a missing space between the two descriptions. Perhaps using a `margin-left` on the second one is better than a space character?

Reply:
Sounds good. Let's try the flex-box approach here as well, but if that is clunky or doesn't look good for any reason, we can use a space.